### PR TITLE
Ensf400_w24_adb added test_always_fails functionality

### PR DESCRIPTION
**Pull request by aaron-dalbroi**

### CODE REVIEW

**Comment by abarcomb-work** [comment]
Pull requests should contain useful information about the reasoning behind the changes for the repository maintainer to enable them to prioritize PRs. This is part of the collaborative workflow.

**Comment by abarcomb-work** [comment]
The PR should have been made to the ensf400-ws24 branch, not to main.

File: tests/baseline_test.py
Comment (growlf) [inline tests/baseline_test.py:1]: This change will cause the CI/CD workflow to _always_ fail, and cannot be accepted as-is.  There are valid reasons to use such, but additional code that uses `assert` to allow for false failures would also be required and is not represented in this commit.

**Comment by growlf** [comment]
Make sure to _not_ check in your `.idea` files as they belong to your local work environment.  Best way to do that is by adding them to the exclusion lists in the `.ignore*` files that are already in the project.

**Pull request closed without merging**